scripts/fetch_target_logs.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+set -x
+
+DIR_NAME="/tmp/target_cluster_logs"
+NAMESPACES="$(kubectl --kubeconfig="${KUBE_CONFIG}" get namespace -o json | jq -r '.items[].metadata.name')"
```

Comment:
Nit. It is not necessary to use `jq` here.
```suggestion
NAMESPACES="$(kubectl --kubeconfig="${KUBE_CONFIG}" get namespace -o jsonpath='{.items[*].metadata.name}')"
```